cli/running/process.go

```diff
@@ -390,6 +390,16 @@ func NewStateboardProcess(ctx *context.Ctx) *Process {
 
 func (process *Process) Log(follow bool, n int) error {
 	if _, err := os.Stat(process.logFile); err != nil {
+		if os.IsNotExist(err) {
+			logFilePathParts := strings.SplitN(process.logFile, "/", -1)
+			logFileName := logFilePathParts[len(logFilePathParts)-1]
+			logFilePathParts[len(logFilePathParts)-1] = strings.SplitN(logFileName, ".", 2)[1]
+			possibleLogFilePath := strings.Join(logFilePathParts, "/")
+			if _, err := os.Stat(possibleLogFilePath); err == nil {
+				return fmt.Errorf("Failed to use process log file: " +
```

Comment:
Can you reuse existing util here too?

Reply:
Hmm, it looks like it can only be moved to `utils`, since the check logic is different here

Reply:
You can wrap this error on the upper level in `cli/running/running.go`. You can do the same for `Status` too (we decided not to wrap this one in the start, but if you implement this in `Log`, implementing this for `Status` would be a 10 seconds task).

Reply:
Done for `Log`

AFAIU, it is not possible to implement this check at the same level for the`Status`

May be it is possible to do this at a lower level (like for `Log` before), but now we have a print:
```
./cartridge status mytestapp.mytestapp                             
   • mytestapp.mytestapp.mytestapp: NOT STARTED
```

I mean, here users can see that there is an extra unnecessary prefix of the specified name before `NOT STARTED`. Do we need a check here?

Reply:
Let's forget about the status then

Reply:
It seems to be even more interesting than that: you can start and stop instances with arbitrary names, see https://github.com/tarantool/cartridge-cli/issues/766

We decided not to cover this issue in this PR